.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
```

Comment:
I can add this to the `.gitignore` if you want to keep the repo IDE agnostic

Reply:
I'm fine with it. Could help any other vscode users out there (like me!).